### PR TITLE
github-management: add madhav as github admin

### PR DESCRIPTION
**Pull request by nikhita**
Ref:

- [sig contribex thread](https://groups.google.com/g/kubernetes-sig-contribex/c/PtLSBDS2yms/m/oFpc3PqUAQAJ) for nomination
- @ameukam's [email](https://groups.google.com/a/kubernetes.io/g/steering/c/-ubWV0Daaxk/m/avoaWI7fAAAJ) to steering to step down 

Lazy consensus achieved.

/committee steering
/cc @kubernetes/steering-committee 
need majority of steering committee to lgtm this change

/assign @cblecker @mrbobbytables @palnabarun @idvoretskyi 
Can you also +1 here?

/assign @MadhavJivrajani 
Can you confirm that you are interested in joining the GitHub Admin Team and accept the security embargo policy?

### CODE REVIEW

**Comment by MadhavJivrajani** [comment]
Thank you @nikhita! 

I confirm that I am interested in joining the GitHub admin team and I accept the security embargo policy.

**Comment by mrbobbytables** [comment]
lgtm with both steering and github admin hats on :)

**Comment by palnabarun** [comment]
Congratulations again, Madhav! 🎉 

/lgtm
(with both steering and GitHub admin hats)

**Comment by tpepper** [comment]
The part adding madhav as approver is very much a
/lgtm
from me with Steering Committee hat on.

I do note that also mixed in with that is the removal of Arnaud and shifting Nabarun from reviewer to approver.  This should probably be a separate commit, but more than the form of commits...the content caught my eye: This change leaves only Ihor on the reviewers list, and his time is obviously pulled to larger concerns.  What's the pipeline looking like for bringing up other contributors in this area?

File: github-management/OWNERS [hunk 1]
```diff
@@ -4,11 +4,11 @@ approvers:
   - cblecker
   - mrbobbytables
   - nikhita
+  - palnabarun
+  - MadhavJivrajani
 
 reviewers:
```
Comment (BenTheElder) [inline github-management/OWNERS:10]: not having the approvers on the reviewers list means only ihor will be assigned reviews which doesn't seem likely to be the intention currently?

**Comment by cblecker** [comment]
lgtm for contribex, gh admin, and steering hats

**Comment by nikhita** [comment]
> I do note that also mixed in with that is the removal of Arnaud and shifting Nabarun from reviewer to approver. 

Nabarun is already a GitHub Admin and it was an oversight to not have him part of approvers earlier.

> not having the approvers on the reviewers list means only ihor will be assigned reviews which doesn't seem likely to be the intention currently?

I've removed `reviewers` from OWNERS entirely. The current list of approvers are the ones actively participating in issues in k/org and the _#github-management_ channel on slack.

> What's the pipeline looking like for bringing up other contributors in this area?

We are periodicially evaluating github admin bandwidth and activity. We will bring in new folks if they have demostrated proactiveness in leading initiatives in github management and are a trusted org member. In fact, this is also how @MadhavJivrajani is being brought on as an admin.

---

With lgtm's from:
- @mrbobbytables 
- @palnabarun 
- @tpepper 
- @cpanato 
- @cblecker 

We have 5/7 steering votes, which satisfies the steering majortity vote criteria. Removing the hold. We can address any future comments about OWNERS as a follow-up.

**Pull request merged**